File: javascript/concepts/operators.js
async function demonstrateOperators() {
  console.log("\n=== Operators ===");
  const a = 10;
  const b = 3;

  // Arithmetic operators
  console.log(`Addition: ${a + b}`);
  console.log(`Subtraction: ${a - b}`);
  console.log(`Multiplication: ${a * b}`);
  console.log(`Division: ${a / b}`);

  // Comparison operators
  console.log(`Is a > b? ${a > b}`);
  console.log(`Is a == b? ${a == b}`);
}

module.exports = demonstrateOperators;

File: golang/concepts/operators.go
package main

import "fmt"

// Operators
func demonstrateOperators() {
	fmt.Println("\n=== Operators ===")
	a := 10
	b := 3

	// Arithmetic Operators
	fmt.Println("Arithmetic Operators")
	fmt.Printf("Addition: %d\n", a+b)
	fmt.Printf("Subtraction: %d\n", a-b)
	fmt.Printf("Multiplication: %d\n", a*b)
	fmt.Printf("Integer Division: %d\n", a/b)
	fmt.Printf("Float Division: %f\n", float64(a)/float64(b))

	// Comparison Operators
	// The '==' operator in Go is already strict (compares type and value).
	fmt.Println("\nComparison Operators")
	fmt.Printf("Is a > b? %t\n", a > b)
	fmt.Printf("Is a == b? %t\n", a == b)
}
